service/update.check/pom.xml

```diff
@@ -88,6 +81,8 @@
                 <configuration>
                     <additionalClasspathElements>
                         <additionalClasspathElement>${basedir}/src/test/resources/gateway.server-5.0.0.8.jar</additionalClasspathElement>
+                        <argLine>${surefireArgLine}</argLine>
+                        <skipTests>${skipTests}</skipTests>
                     </additionalClasspathElements>
                 </configuration>
             </plugin>
```

Comment:
@dpwspoon there is a warning in the Travis build related to this plugin not having a version specified, not sure how to get around it(just add a version? shouldn't it inherit the version?); also why is the special surefire configuration needed for this plugin?